csi/server/examples/kubernetes/nginx.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   name: csi-sc-opensdsplugin
 provisioner: csi-opensdsplugin
 parameters:
-
+  fsType: ext4
```

Comment:
Is this needed for raw block mode?

Reply:
This has nothing to do with the raw block, just to ensure that the basic example is successful.